client/components/tinymce/index.jsx

```diff
@@ -380,7 +381,10 @@ module.exports = React.createClass( {
 
 			// Collapse selection to avoid scrolling to the bottom of the textarea
 			textNode.setSelectionRange( 0, 0 );
-			textNode.focus();
+
+			if ( ! isIE11Detected ) {
```

Comment:
To me, I think `isIE11` is a good enough variable name. But no worries if you like it as-is